modules/ztp-sno-du-configuring-ptp.adoc

```diff
@@ -8,8 +8,27 @@
 
 {sno-caps} clusters use Precision Time Protocol (PTP) for network time synchronization. The following example `PtpConfig` CR illustrates the required PTP slave configuration.
 
-.Recommended PTP configuration
+.Recommended PTP ordinary clock configuration (`PtpConfigSlave.yaml`)
 [source,yaml]
 ----
 include::snippets/ztp_PtpConfigSlave.yaml[]
 ----
+
+
+.Recommended boundary clock configuration (`PtpConfigBoundary.yaml`)
+[source,yaml]
+----
+include::snippets/ztp_PtpConfigBoundary.yaml[]
+----
+
+.Recommended PTP Westport Channel e810 grandmaster clock configuration (`PtpConfigGmWpc.yaml`)
+[source,yaml]
+----
+include::snippets/ztp_PtpConfigGmWpc.yaml[]
+----
+
+.Recommended PTP events configuration (`PtpOperatorConfigForEvent.yaml`)
```

Comment:
This one is optional. The other PtpConfig* are "pick one of these". Not sure if that needs to be mentioned?